src/otaclient/ota_core/_updater.py

```diff
@@ -211,13 +211,21 @@ def _execute_update(self):
     def _pre_update(self):
         """Prepare the standby slot and optimize the file_table."""
         logger.info("enter local OTA update...")
+
+        # NOTE(20250905): if ota_resources dir on active slot presented,
+        #                 no need to use rebuild mode.
+        _ota_resources_dir_presented = self._resource_dir_on_active.is_dir()
```

Comment:
Could you elaborate why we don't have to use rebuild mode when there is ota_resources dir on active slot? In my understanding,
if there is `_resource_dir_on_active`, we can copy active resource_dir from active to standby in in-place mode. So we don't have to do rebuild mode, is my understanding correct?

https://github.com/tier4/ota-client/blob/c00c0699e4c9ed1d72bace5049885b07b4f6d3dd/src/otaclient/ota_core/_updater.py#L418-L427